src/main/api/opds.ts

```diff
@@ -77,6 +87,12 @@ export class OpdsApi {
                 return opdsFeedData;
             }
 
+            debug("opdsFeed content-type", opdsFeedData.contentType);
+            if (!OpdsApi.contentTypeisAccepted(opdsFeedData.contentType)) {
+                throw new Error(`Not a valid OPDS HTTP Content-Type:
+                    ${opdsFeedData.url} => ${opdsFeedData.contentType}`);
```

Comment:
The line break and other whitespaces are literal (backticks string), resulting in `\n`. See screenshot:

![ThoriumOPDSfail](https://user-images.githubusercontent.com/625201/63962229-5f056f00-ca8a-11e9-9e81-e472cd014257.png)

Reply:
i saw

Reply:
done

I added the timeout, it's was went out